docs/api/XdpGuidCreate.md

```diff
@@ -0,0 +1,21 @@
+# XdpGuidCreate function
+
+Initializes a GUID.
+
+## Syntax
+
+```C
+_IRQL_requires_max_(PASSIVE_LEVEL)
+NTSTATUS
+XdpGuidCreate(
```

Comment:
Why does XDP need its own function for this?

Reply:
Looks like it's a shim for `ExUuidCreate` that ignores failures when the GUID might only be LUID.

```
    Status = ExUuidCreate(Guid);
    if (Status == RPC_NT_UUID_LOCAL_ONLY) {
        //
        // The GUID is only a LUID, which is perfectly fine.
        //
        Status = STATUS_SUCCESS;
    }
```

This is to simplify native driver development and avoid pitfalls.

Reply:
Ok.